server/services/prismic.js

```diff
@@ -97,9 +104,13 @@ export async function getEvent(id: string, previewReq: ?Request): Promise<?Event
   const fetchLinks = eventFields.concat(peopleFields, contributorFields, seriesFields);
   const event = await getTypeById(previewReq, ['events'], id, {fetchLinks});
 
+  const scheduleIds = event.data.schedule.map(event => event.event.id);
+
+  const eventScheduleDocs = scheduleIds.length > 0 && await getTypeByIds(previewReq, ['events'], scheduleIds, {fetchLinks});
```

Comment:
This breaks the idea of 1 request / render - could we please change that?
Or make it more explicit somewhere?